godtools/App/Services/Renderer/Views/Tool/Views/Cards/ToolPageCardsView.swift

```diff
@@ -353,6 +353,8 @@ extension ToolPageCardsView {
             cardTopConstraints.append(top)
             
             cardView.setDelegate(delegate: self)
+            
+            cardView.layoutSubviews()
```

Comment:
Hey @reldredge71 one last thing, I'm pretty sure you don't need this call here. layoutSubviews() will get called by the system when the constraints layout. 
You also don't want to call layoutSubviews() directly according to Apple's documentation.  Instead you'd want to call either setNeedsLayout() or layoutIfNeeded() depending on your needs.  https://developer.apple.com/documentation/uikit/uiview/1622482-layoutsubviews